lib/ImagePackager.js

```diff
@@ -20,7 +19,7 @@ class ImagePackager extends RawPackager {
       contents = Buffer.from(contents.data);
     }
 
-    await mkdirp(path.dirname(name));
+    await fs.ensureDir(path.dirname(name));
     
     await new Promise((resolve, reject) => {
```

Comment:
Change this to use fs-extra as well, no need to wrap something that returns a promise in a promise :)